### PR TITLE
publiccloud: ajust pc_tools.xml yast to SP3

### PR DESCRIPTION
**Pull request by ilausuch**
During the tests to addapt pc tools to SP3 there where problems
during the installation step in tests caused by pc_tools.xml

- Related ticket: https://progress.opensuse.org/issues/105211
- Verification run: http://copland.arch.suse.de/tests/744

### CODE REVIEW

**Review by pdostal — APPROVED** [review]
Good to go :rocket:

**Comment by jlausuch** [comment]
Just to point out the profile differences (besides the tabs), 15-SP3 didn't allow the following lines of the profile:
```
                <ask_user config:type="boolean">false</ask_user>
                <selected config:type="boolean">true</selected>
```

so, they have been removed.

```
jing /usr/share/YaST2/schema/autoyast/rng/profile.rng pc_tools.xml  
/var/lib/openqa/share/tests/opensuse/data/autoyast_sle15/pc_tools.xml:30:49: error: element "ask_user" not allowed here; expected the element end-tag or element "signature-handling"
/var/lib/openqa/share/tests/opensuse/data/autoyast_sle15/pc_tools.xml:31:49: error: element "selected" not allowed here; expected the element end-tag or element "signature-handling"
```

**Review by jlausuch — APPROVED** [review]

**Comment by ilausuch** [comment]
Works together with https://gitlab.suse.de/qac/qac-openqa-yaml/-/merge_requests/588

**Comment by pdostal** [comment]
/Ajust/Adjust/

File: data/autoyast_sle15/pc_tools.xml [hunk 1]
```diff
@@ -1,146 +1,144 @@
...
-            <user_password>nots3cr3t</user_password>
-            <username>bernhard</username>
-        </user>
-        <user>
-            <encrypted config:type="boolean">false</encrypted>
-            <user_password>nots3cr3t</user_password>
-            <username>root</username>
-        </user>
-    </users>
+  <general>
```
Comment (asmorodskyi) [inline data/autoyast_sle15/pc_tools.xml:4]: not sure why similar lines are marked in the diff . it is hard to see what actually was changed ...

Comment (jlausuch) [reply data/autoyast_sle15/pc_tools.xml:4]: Looks like the 4 spaces were converted into 2 spaces, same as all the other profiles we have.
Anyway, the main difference is these 2 lines:
```
<ask_user config:type="boolean">false</ask_user>
<selected config:type="boolean">true</selected>
```
which are not compatible with 15-SP3.

Comment (ilausuch) [reply data/autoyast_sle15/pc_tools.xml:4]: As suggestion of @asmorodskyi I split the indentation cosmetic in an other commit to simplify the review

**Pull request merged**